model/src/main/java/com/basistech/rosette/dm/AnnotatedText.java

```diff
@@ -47,18 +48,19 @@
  * For Json, and Java, the 'adm-json' module provides the supported serialization.
  */
 @SuppressWarnings("deprecation")
-public class AnnotatedText {
+public class AnnotatedText implements Serializable {
+    private static final long serialVersionUID = 222L;
```

Comment:
Oh, 222 is from the version number!  Okay.  Could we document that convention somewhere?